app/assets/stylesheets/components/_index.scss

```diff
@@ -10,4 +10,5 @@
 @import "navbar";
 @import "_outfit_card";
 @import "_form";
-@import "_outfit_new"
+@import "_outfit_new";
```

Comment:
For future - be more specific with the title of the component. outfit_new_card would be more descriptive!